bin/setup-bench.mjs

```diff
@@ -1,7 +1,10 @@
-import 'zx/globals';
+/* eslint-disable n/no-process-exit */
+import { readFile, writeFile } from 'node:fs/promises';
 import os from 'node:os';
 import { join } from 'node:path';
-import { readFile, writeFile } from 'node:fs/promises';
+
+import { ensureDirSync, writeFileSync } from 'fs-extra';
```

Comment:
the default import for fs-extra is actually cjs, so you have to do:

```js
import fse from 'fs-extra';

const { ensureDirSync, writeFileSync } = fse;

```

Reply:
Ooh good point!